app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    """Copies the contents of one file to another"""
+
+    # we check the command structure
```

Comment:
Please, remove the redundant comment

Reply:
Don't leave comments in the final version of the code
Code should self-descriptive enough

Reply:
please, fix everywhere